test/oc_trace_context_SUITE.erl

```diff
@@ -12,91 +12,94 @@
 -include("opencensus.hrl").
 
 all() ->
-    [encode_decode, decode_with_extra_junk, encode_decode_headers].
+    [encode_decode,  decode_with_extra_junk,  encode_decode_headers].
```

Comment:
Where did all these extra spaces in this file come from? :)

Reply:
ah sneaky stuff, it came from trying to add missing spaces ;) I thought I gut them all -.- sorry